package/nodejs/nodejs.mk

```diff
@@ -4,19 +4,20 @@
 #
 ################################################################################
 
-NODEJS_VERSION = 12.18.0
+NODEJS_VERSION = 14.18.3
```

Comment:
NodeJS 14.X is right now in the maintenece mode, have you tried to use version 16.X? It's currently active LTS

Reply:
I ported 14.18.3 from buildroot upstream repo: https://github.com/buildroot/buildroot/blob/master/package/nodejs/nodejs.mk. Will move to 16.X once it is bumped up by the upstream Buildroot repo